hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -338,6 +341,7 @@ protected HStore(final HRegion region, final ColumnFamilyDescriptor family,
         getColumnFamilyName(), memstore.getClass().getSimpleName(), policyName, verifyBulkLoads,
         parallelPutCountPrintThreshold, family.getDataBlockEncoding(),
         family.getCompressionType());
+    isStoreFileForceArchivalEnabled = this.conf.getBoolean(FORCE_ARCHIVAL_AFTER_RESET, false);
```

Comment:
Let the CP do this setter on this StoreScanner that it is wrapping . So that it is specific to that scanner only?

Reply:
Sure, we can set this `false` by default as is and let CP set it to true. Override of this config value by CP should be taken care of by this line.